test/Microsoft.AspNetCore.Hosting.Tests/WebHostBuilderTests.cs

```diff
@@ -319,6 +319,42 @@ public void RelativeApplicationBaseAreResolved()
             Assert.EndsWith(Path.DirectorySeparatorChar + "bar", basePath);
         }
 
+        [Fact]
+        public void DefaultApplicationNameToStartupAssemblyName()
```

Comment:
What do you mean test the setting? This version of `UseStartup` calls `UseSetting`.